src/main/java/spireMapOverhaul/abstracts/AbstractZone.java

```diff
@@ -211,8 +211,15 @@ public void renderOnMap(SpriteBatch sb, float alpha) {
             }
             float anchorX = x * SPACING_X + OFFSET_X - ZoneShapeMaker.FB_OFFSET;
             float anchorY = y * Settings.MAP_DST_Y + OFFSET_Y + DungeonMapScreen.offsetY - ZoneShapeMaker.FB_OFFSET;
-            sb.setColor(getAdjustedColor().cpy().mul(1f, 1f, 1f, alpha*0.5f));
+
+            Color oldColor = sb.getColor().cpy();
```

Comment:
```suggestion
            Color oldColor = sb.getColor()
```
No need to make a copy if we're just going to slot it back in after